manifests/05-operator.yaml

```diff
@@ -41,6 +41,13 @@ spec:
           value: docker.io/openshift/origin-console:latest
         - name: OPERATOR_NAME
           value: "console-operator"
+        resources:
```

Comment:
So this manifest is for the `console-operator`, not the `console`.  I haven't thought about it, but the `console-operator` may need resource limits as well (?).

The `console` needs this block in `pkg/console/subresources/deployment/deployment.go`.  It's going to be added via the `golang` code as the manifest is generated. I think its around `line 180` I have a `TODO` marked where it can be stubbed in.

Reply:
I'd think we'd want values for both the operator and the console so we're not running best effort pods.